src/libData/AccountData/AccountStore.cpp

```diff
@@ -95,7 +95,7 @@ int AccountStore::Deserialize(const vector<unsigned char>& src,
             UpdateStateTrie(address, account);
             // MoveUpdatesToDisk();
         }
-        PrintAccountState();
+        // PrintAccountState();
```

Comment:
If these are good for debugging they should be protected by a macro e.g. `DEBUG(codeBlock)`